app/models/mixins/authentication_mixin.rb

```diff
@@ -226,7 +255,21 @@ def authentication_check_types(*args)
     # Let the individual classes determine what authentication(s) need to be checked
     types = authentications_to_validate if self.respond_to?(:authentications_to_validate) && types.nil?
     types = [nil] if types.blank?
-    types.to_miq_a.each { |t| authentication_check(t, options) }
+    types.to_miq_a.each do |t|
+      success = authentication_check(t, options).first
+      retry_scheduled_authentication_check(t, options) unless success
+    end
+  end
+
+  def retry_scheduled_authentication_check(t, options)
+    return unless options[:attempt]
+    auth = authentication_best_fit(t)
+    if auth.retryable_status?
+      options[:attempt] += 1
+
+      # Force the authentication message to be queued
+      authentication_check_types_queue(t, options.merge(:force => true))
```

Comment:
Normally, this method would skip queueing the message because there already is one, the one we're working on.  Since, we want to retry this one, pass the force option.